hudi-utilities/src/main/java/org/apache/hudi/utilities/UtilHelpers.java

```diff
@@ -102,11 +104,23 @@ public static SchemaProvider createSchemaProvider(String schemaProviderClass, Ty
     }
   }
 
-  public static Transformer createTransformer(String transformerClass) throws IOException {
+  public static Transformer createTransformer(String transformerClasses) throws IOException {
```

Comment:
Can we add a CLI test case for arg `--transformer-class` so that we can cover the change of this method?

Reply:
ok done.